src/starterBricks/sidebarExtension.ts

```diff
@@ -342,10 +330,9 @@ export abstract class SidebarStarterBrickABC extends StarterBrickABC<SidebarConf
 
   /**
    * Shared event handler for DOM event triggers.
+   * It's bound to this instance so that it can be removed when the extension is uninstalled.
```

Comment:
Is this what you mean in our newest terminology?
```suggestion
   * It's bound to this instance so that it can be removed when the mod is deactivated.
```

Reply:
Maybe? The file is named `-Extension` and the class has a `uninstall` method, so we should probably match those (one way or another)

Reply:
Part of the continuing naming/domain modeling work is to move from installed/uninstalled -> activated/deactivated. 

We haven't really discussed changing the class methods though.

Reply:
I'll update the comment for now because that's more likely to be left behind when the methods change.